src/Api/techApi.ts

```diff
@@ -0,0 +1,45 @@
+import axios from "axios";
+import { techs_url } from "./urls";
+import {
+  IApiResponse,
+  getRequestConfig,
+  axiosHandleResponse,
+} from "./utilsApi";
+
+export interface Tech {
+  id: string;
+  name: string;
+}
+export interface Techs {
+  techs: Tech[];
+}
+export interface NewTech {
+  name: string;
+  passworld: string;
```

Comment:
passworld, the world of passes :P

Reply:
To coś jest źle bo zmieniłem na ...word bo bez tego backend nie dziala